pkg/agent/nodeportlocal/rules/iptable_rule.go

```diff
@@ -151,3 +148,16 @@ func (ipt *IPTableRules) DeleteAllRules() error {
 	}
 	return nil
 }
+
+// Join all words with spaces, terminate with newline and write to buf.
+func writeLine(buf *bytes.Buffer, words ...string) {
```

Comment:
if this function is the same as the one in pkg/agent/route/route_linux.go, it should got into a common utility package, but that can be done in a future PR

Reply:
Yeah, noted. Will take care of it in a subsequent PR.